Library/Formula/moose.rb

```diff
@@ -0,0 +1,60 @@
+class Moose < Formula
+  desc "MOOSE is a Multiscale Object Oriented Simulation Environment for simulating multiscale neural systems."
+  version "3.0.2"
+  homepage "http://moose.ncbs.res.in"
+  url "https://github.com/BhallaLab/moose-core/archive/ghevar-3.0.2-alpha.zip"
+  sha256 "febc7fc41a2513aeb2b6998731f61c7e7f2a6b5169a77e1612c189779fb0d1f1"
+
+  depends_on "cmake" => :build
+  depends_on "gsl" 
+  depends_on "homebrew/science/hdf5" 
+  depends_on "homebrew/science/libsbml" 
+  depends_on "homebrew/python/matplotlib"
+  depends_on "homebrew/python/numpy"
+  depends_on "python"
```

Comment:
Should be `depends_on :python if MacOS.version <= :snow_leopard` presuming the reason for the added dep is the necessity of 2.7. We don't unconditionally enforce our Python.